README.md

```diff
@@ -18,38 +18,115 @@ A set of conventions for local AWS Lambda software development.
 ├── cloudformation                     # CloudFormation template and parameters.
 │   └── parameters.json
 │   └── template.yaml
+├── config.yaml                        # Static config and ParameterStore lookups.
 ├── index.py                           # Entry point for the Lambda function.
-├── my_lambda_package                  # Python package `my_lambda_package`.
-│   ├── __init__.py
-│   ├── localcontext.py
-│   ├── utility.py
 ├── requirements                       # External dependencies.
 │   ├── common.txt
 │   ├── dev.txt
 │   └── lambda.txt
 └── tests                              # Unit tests for the package.
     ├── __init__.py
-    └── my_lambda_package
+    └── utils
         ├── __init__.py
         ├── test_localcontext.py
-        └── test_utility.py
+        └── test_helpers.py
+├── utils                              # Python package `utils`.
+│   ├── __init__.py
+│   ├── config.py
+│   ├── localcontext.py
+│   ├── helpers.py
 ```
 
 ## 1. Initial AWS Setup
 
 Creates a CloudFormation stack with the Lambda function, an execution role, and an optional CloudWatch event to run on a recurring basis.
 
+### A) Stack Naming and Parameters
+
+Edit `cloudformation/parameters.json`, and supply appropriate parameters.
+
+In particular, Select appropriate values for `ProjectName` and `EnvironmentName` in `cloudformation/parameters.json`.
+
+**Important: The resulting CloudFormation stack will be named `${ProjectName}-${EnvironmentName}-stack`, and a stack name of this form will be presumed for future CloudFormation operations.**
+
+### B) Permissioning
+
+Edit `cloudformation/template.yaml` and ensure that the Lambda function is appropriately permissioned via the policies attached to the `LambdaFunctionExecutionRole`.
+
+### C) Optional: Set Values in Parameter Store
```

Comment:
This would work better as CFN + parameters.json, but it's a good MVP.

Reply:
Setting these parameters via CFN is what I mean by "work better", rather than having to run a bash script manually.